ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -457,7 +457,7 @@ private static void createLine(UcteNetworkExt ucteNetwork, Network network,
                 createDanglingLine(ucteLine, connected, xnode, nodeCode1, ucteVoltageLevel1, network);
 
             } else {
-                throw new UcteException("Line between 2 Xnodes");
+                throw new UcteException("Line between 2 Xnodes: '" + nodeCode1 + "' and '" + nodeCode2 + "'");
```

Comment:
Just a detail but in the rest of the code, in error messages we write "X-node". I don't know what way is the good one.

Reply:
In UCTE import, I can see one "Xnode" and one "XNODE"

Reply:
What do you prefer? I will commit in your PR just a uniformisation if you don't mind.

Reply:
I replaced all by X-node